src/infrastructure/note.repository.ts

```diff
@@ -1,11 +1,22 @@
 import NoteRepositoryInterface from '../domain/note.repository.interface';
 import Note from '../domain/entities/Note';
 import NoteStorage from './storage/note';
+import NotesApiTransport from './transport/notes-api';
+
+/**
+ * Get note response payload
+ */
+type GetNoteResponsePayload = Note;
```

Comment:
move it to the `transport/note-api/types/`

Reply:
done